lib/dbservice/batches/batch.ex

```diff
@@ -15,6 +16,7 @@ defmodule Dbservice.Batches.Batch do
 
     has_many :group_type, GroupType, foreign_key: :child_id, where: [type: "batch"]
     many_to_many :program, Program, join_through: "batch_program", on_replace: :delete
+    has_one :session_schedule, SessionSchedule
```

Comment:
This looks so nice :) Batch has one session schedule. Makes sense in English also :)